tests/integrations/subprocess/test_Subprocess__final_kwargs.py

```diff
@@ -47,10 +48,22 @@ def test_env_overrides(mock_sub):
 
 def test_cwd_provided(mock_sub):
     """If a cwd is provided, it is reflected in the environment."""
+    cwd_override = "/my/current/path"
+    assert mock_sub.final_kwargs(cwd=cwd_override) == {
+        "text": True,
+        "encoding": CONSOLE_ENCODING,
+        "cwd": cwd_override,
+    }
```

Comment:
I was clearly having a good day when I approved this tests the first time around...